src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const items = document.querySelectorAll('.population');
+const numbers = [...items].map(item => +item.innerText.split(',').join(''));
+const total = numbers.reduce((a, b) => a + b);
+const avarage = total / numbers.length;
+
+const totalStr = total.toString().split('');
+
+totalStr.splice(-3, 0, ',');
+totalStr.splice(-7, 0, ',');
+totalStr.splice(-11, 0, ',');
+
+const totalResult = totalStr.join('');
```

Comment:
try `toLocaleString`, this can reduce the amount of your code

Reply:
thank you! I didn't know about such a method))